var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -220,19 +227,26 @@ def cmake_args(self):
 
         # Core options
         options = [
-            self.define("GEANT4_USE_SYSTEM_CLHEP", True),
             self.define("GEANT4_USE_SYSTEM_EXPAT", True),
             self.define("GEANT4_USE_SYSTEM_ZLIB", True),
             self.define("GEANT4_USE_G3TOG4", True),
             self.define("GEANT4_USE_GDML", True),
             self.define("XERCESC_ROOT_DIR", spec["xerces-c"].prefix),
         ]
 
+        if "+builtin_clhep" in spec:
```

Comment:
We generally recommend use of `spec.satisfies()` for checking variants for reasons I'll paste below.  I realize in this case the odds of other packages having this variant are pretty low ATM so this is more of an FYI.

Per https://spack.readthedocs.io/en/latest/packaging_guide.html#testing-spec-constraints:

"The difference between the two functions is that `satisfies()` tests whether spec constraints overlap at all, while `in` tests whether a spec or any of its dependencies satisfy the provided spec."